layout-server/src/test/java/org/finos/vuu/layoutserver/controller/ApplicationLayoutControllerTest.java

```diff
@@ -0,0 +1,65 @@
+package org.finos.vuu.layoutserver.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.finos.vuu.layoutserver.dto.response.ApplicationLayoutDto;
+import org.finos.vuu.layoutserver.model.ApplicationLayout;
+import org.finos.vuu.layoutserver.service.ApplicationLayoutService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.modelmapper.ModelMapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+class ApplicationLayoutControllerTest {
+    private static ApplicationLayoutService mockService;
+    private static ApplicationLayoutController controller;
+    private static final ModelMapper modelMapper = new ModelMapper();
```

Comment:
This is an interest point. I think I'd prefer not to mock it in this case, because we're not writing unit tests for the mapper itself. That makes sense because it's an external dependency that we're treating as a utility (in my mind, this is no different to using an inbuilt Java function - you wouldn't mock the result of an operation on a date/time, for example). I suppose it is made slightly more complex by the fact that `ModelMapper` is configurable though.